docs/tutorial/02-draw-a-snake.md

```diff
@@ -67,7 +67,7 @@ We turn our module into a Scenic "scene" by using `Scenic.Scene`. This will auto
 
 ```elixir
 def init(arg, opts) do
-
+  {:ok, :ok}
```

Comment:
I Scenic complaining when we're skipping that part? I didn't notice 🤔

Reply:
Yeah, at least I get an error screen instead of the black screen

Reply:
Ok. Then we should stick with returning something 👍